### PR TITLE
sd-bus: Add sd_bus_query_sender_creds/privilege docs

### PR DESCRIPTION
**Pull request by daandemeyer**
Last ones for sd-bus!

### CODE REVIEW

File: man/sd_bus_query_sender_creds.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        <paramdef>int <parameter>capability</parameter></paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_query_sender_creds()</function> returns the credentials of the
+    <parameter>call</parameter> message. The <parameter>mask</parameter> parameter is a combo of
```
Comment (keszybz) [inline man/sd_bus_query_sender_creds.xml]: Hmm, maybe rename this (in the sources too) to `message`? This doesn't only apply to method calls.

Comment (daandemeyer) [reply man/sd_bus_query_sender_creds.xml]: Renamed to `m` since `message` is not used in the sd-bus.h header

File: .clang-format [hunk 2]
```diff
@@ -28,7 +27,7 @@ BreakBeforeBraces: Custom
 BreakInheritanceList: BeforeComma
 BreakBeforeTernaryOperators: false
 BreakStringLiterals: false
-ColumnLimit:     109
+ColumnLimit: 0
```
Comment (keszybz) [inline .clang-format]: Hmm, are those changes to `.clang-format` by mistake?

Comment (daandemeyer) [reply .clang-format]: My bad, good catch!

**Pull request merged**